src/styles/blocks/card.scss

```
@@ -0,0 +1,95 @@
+.card {
+  width: $card-width;
+  padding: 0 16px;
+
+  border: 1px solid #e5e5e5;
+  border-radius: 5px;
+
+  &__img {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: auto;
+    margin-top: 32px;
+  }
+
+  &__title {
+    margin-top: 40px;
+
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+
+    color: $text-color-main;
+  }
+
+  &__product-code {
+    margin-top: 4px;
+
+    font-size: 10px;
+    line-height: 14px;
+
+    color: $text-color-secondary;
+  }
+
+  &__rating {
+    margin-top: 16px;
+    display: flex;
+    justify-content: space-between;
+
+    font-size: 10px;
+    line-height: 14px;
+
+    color: $text-color-main;
+  }
+
+  &__price {
+    margin-top: 24px;
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &__price-name {
+    font-size: 12px;
+    line-height: 18px;
+
+    color: $text-color-secondary;
+  }
+
+  &__price-value {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+
+    color: $text-color-main;
+  }
+
+  &__button {
+    margin: 16px 0;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    width: 100%;
+    height: 40px;
+
+    border: 1px solid tranparent;
+    border-radius: 5px;
+    text-decoration: none;
+
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    text-transform: uppercase;
+
+    background-color: $color-accent;
+    color: #fff;
```

You create variables for colors, but not for all 🥺 
```suggestion
    color: $c-white;
```